### DISCUSSION TITLE
First early-release fails for tag deletion

### DISCUSSION THREAD

**Discussion by manikmagar** [jreleaser/jreleaser#1738] [General]
Hi! 

We upgraded to use 1.14.0, and we ran into this error.

This is a new[ test repo project](https://github.com/avioconsulting/jreleaser-test-app/blob/main/jreleaser.yml); its first early release, but it is failing to delete a non-existent early-access, tag. Am I missing any configuration that may have changed in new release?

```
[INFO]    [upload] Uploading is not enabled. Skipping
[INFO]  Releasing to https://github.com/avioconsulting/jreleaser-test-app@main
[DEBUG] fetching all branches on avioconsulting/jreleaser-test-app
[DEBUG] looking up release with tag early-access at repository avioconsulting/jreleaser-test-app
[DEBUG] fetching release on avioconsulting/jreleaser-test-app with tag early-access
[DEBUG] release early-access does not exist
[DEBUG] creating release early-access
[DEBUG] deleting tag early-access from avioconsulting/jreleaser-test-app
jreleaser.shadow.org.jreleaser.sdk.commons.RestAPIException: 404: Not Found
{"message":"Not Found","documentation_url":"https://docs.github.com/rest/git/refs#delete-a-reference","status":"404"}
	at jreleaser.shadow.org.jreleaser.sdk.commons.ClientUtils.lambda$builder$1(ClientUtils.java:120)
	at jreleaser.shadow.feign.InvocationContext.decodeError(InvocationContext.java:126)

```

**Comment by aalmiray** [comment]
Perhaps GitHub updated their constraints recently as we have not changed the code that tags and publish a new repository in a while. 

However, it seems JReleaser should be able to handle this kind of error. Could you please file a bug referencing this discussion? Thanks.

**Reply by manikmagar** [reply]
Done. Please see #1739

**Reply by manikmagar** [reply]
I am going to try using previous versions of the JReleaser in the same test app now.
